### PR TITLE
#49 layout attributes for supplementary items

### PR DESCRIPTION
**Pull request by christoff-1992**
This resolves the crash described in #49, currently the headers attributes are added per number of items in the section causing conflicts due to the inconsistent sizes when cells are not visible.

The attributes should only be added once per section which the other layouts already do.

### CODE REVIEW

**Comment by christoff-1992** [comment]
@zenangst - ahh I keep forgetting about the other test suites in the other targets! will get them updated.

**Comment by zenangst** [comment]
@christoff-1992 pushed a small update to fix the failing tests ;)

File: Sources/Shared/Core/HorizontalBlueprintLayout.swift [hunk 1]
```diff
@@ -147,12 +148,12 @@
 
           if stickyHeaders {
             headerAttribute?.frame.origin.x = headerFooterX
-            headerAttribute?.frame.size.width = min(headerFooterWidth, widthOfSection)
+            headerAttribute?.frame.size.width = max(headerFooterWidth, widthOfSection)
```
Comment (zenangst) [inline Sources/Shared/Core/HorizontalBlueprintLayout.swift]: @christoff-1992 you reckon that this change makes sense?
It does in my head... that the largest of the two should be used. The header should always stretch to the end of the section right? Or perhaps it shouldn't now that I think of it. Perhaps it should actually be `min` here if you are using sticky headers. Then it should simply move across the screen and not span the entire section.

Comment (zenangst) [reply Sources/Shared/Core/HorizontalBlueprintLayout.swift]: I reverted this change and made some updates in 
https://github.com/zenangst/Blueprints/pull/58/commits/2148166afec3029905c19709c9211ebff9615fe0

**Comment by zenangst** [comment]
Let's merge this sucker and start looking into sticky headers :)

**Pull request merged**